api.go

```diff
@@ -656,6 +656,26 @@ func keyHandler(w http.ResponseWriter, r *http.Request) {
 	keyName := mux.Vars(r)["keyName"]
 	apiID := r.URL.Query().Get("api_id")
 	isHashed := r.URL.Query().Get("hashed") != ""
+	isUserName := r.URL.Query().Get("username") == "true"
+
+	// check if passed key is user name and convert it to real key with respect to current hashing algorithm
+	if r.Method != http.MethodPost && isUserName {
```

Comment:
I do understand why it was done like this, however, I feel bad while looking at it.

I propose to make it simple, and in such case, just require passing one more attribute "org_id".

Reply:
do you mean more attribute "org_id" in addition to "username=true" ?